jobs/build/build-sync/Jenkinsfile

```diff
@@ -192,13 +192,17 @@ node('covscan') {
         if (params.SKIP_MULTI_ARCH_PAYLOAD) {
             cmd << "--skip-multiarch-payload"
         }
+        if (params.TRIGGER_BUILD_SYNC) {
```

Comment:
This is misleading, would be clearer if named like `TRIGGERED_FROM_GEN_ASSEMBLY`

Reply:
Fixed in https://github.com/openshift-eng/aos-cd-jobs/pull/3605/commits/43560fc58fd4e127fa655bd82dda758d8965b529